test/experiments_ClimaCore.jl

```diff
@@ -0,0 +1,73 @@
+# Experiments
+
+push!(LOAD_PATH, joinpath(@__DIR__, "../experiments/ClimaCore/tc4_atm-lnd-sfc-fluxes"))
+
+using Test
+
+include(joinpath(@__DIR__,"../experiments/ClimaCore/tc4_atm-lnd-sfc-fluxes/experiment.jl"))
+
+
+FT = Float64
+parameters = (
+        # timestepping parameters 
+        Δt_min = 0.01, # minimum model timestep [s]
+        timerange = (0.0, 1.0),  # start time and end time [s]
```

Comment:
I shortened this time so the test completes more quickly; @LenkaNovak is there a minimum time we want to run this for in the CI?

Reply:
Sounds good. Since this is only an integration test, 100 time steps should be enough :).